src/locales/en.json

```diff
@@ -92,7 +92,7 @@
     "heading": "RECOMMENDED LICENSE",
     "heading-cc0": "RECOMMENDED",
     "full-description": {
-      "cc0": "By marking your work with a CC0 public domain dedication, you are giving up your copyright and allowing reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
+      "cc0": "This license is marked with a CC0 public domain dedication that the user is giving up his copyright and allowing reusers to distribute, remix, adapt, and build upon the material in any medium or format, even for commercial purposes.",
```

Comment:
I would suggest the following changes:
'By marking your work' - 'By marking the work'
'you are' - 'the creator is'
'your' - 'their'. 
@zackkrida , @annatuma , What do you think?